python/ml4ir/applications/ranking/model/metrics/helpers/metrics_helper.py

```diff
@@ -143,7 +143,12 @@ def get_grouped_stats(
     df = add_top_graded_relevance_record_column(df, label_col, artificial_click_col)
     if np.array([Metric.NDCG in m for m in power_analysis_metrics]).any():
         df = compute_ndcg(df, label_col, pred_col="ranking_score", new_col=RankingConstants.NEW_NDCG)
-        df = compute_ndcg(df, label_col, pred_col="s", new_col=RankingConstants.OLD_NDCG)
+        if "s" in df.columns:
```

Comment:
I received a mail, I can not follow all the amazing work we do! For someone who has not enough context,  I can not wrap my head around this `"s" in df.columns`. Is there any change we have a more descriptive condition? If not, perhaps an inline comment would help for the folks who read the code.

Reply:
Sure, adding a comment. 's' is the old model ranking score.

Reply:
Thanks! BTW my suggestions are non-blocking, feel free to merge, I know that I am asking it tough and decisions need to be made timely so that work advances. 

PITA: I just hope that it is clear that this is not a very good comment for an open-source project. 
```
# s is the old model(Prod model at the time of data collection) ranking score    
```
old vs new model and production model for someone outside your team do not help much.

Reply:
If you get to this again, please add a space between `model(Prod `

Reply:
Ping ^

Reply:
@mohazahran You need to get rid off internal company lingo. Like `s`.

Reply:
done